payload/dev/solr/entrypoint.bash

```diff
@@ -9,11 +18,30 @@ if [ ! -f /ezsolr/server/ez/solr.xml ]; then
     sed -i -e 's/<dataDir>${solr.data.dir:}<\/dataDir>/<dataDir>\/opt\/solr\/data\/${solr.core.name}<\/dataDir>/g' /ezsolr/server/ez/template/solrconfig.xml
 fi
 
-/opt/solr/bin/solr -s /ezsolr/server/ez -f
-
 SOLR_CORES=${SOLR_CORES:-collection1}
+CREATE_CORES=false
+
 for core in $SOLR_CORES
 do
-    /opt/solr/bin/solr create_core -c ${core}  -d /ezsolr/server/ez/template
-    echo "Core ${core} created."
-done
\ No newline at end of file
+    if [ ! -d /ezsolr/server/ez/${core} ]; then
+        CREATE_CORES=true
+        echo "Found missing core: ${core}"
+    fi
+done
+
+if [ "$CREATE_CORES" = true ]; then
+    echo "Try to start solr on background..."
```

Comment:
Hello @xserna thanks for that PR. I wonder here way the "Try" did you experience some race conditions or something here? Just wondering.

Thanks!